database.txt

```diff
@@ -1,38 +0,0 @@
- +-------------+      +-----------------+   +---------------+   +------------+
- | users       |      | preferences     |   | filters       |   | rules      |
```

Comment:
It was likely made by hand using asciiflow I suspect.

Nice tool, less nice looking but can be handy in more cases